src/react/components/pages/projectSettings/projectSettingsPage.test.tsx

```diff
@@ -1,16 +1,23 @@
 import { mount, ReactWrapper } from "enzyme";
 import React from "react";
 import { Provider } from "react-redux";
+import { AnyAction, Store } from "redux";
 import { BrowserRouter as Router } from "react-router-dom";
 import MockFactory from "../../../../common/mockFactory";
 import createReduxStore from "../../../../redux/store/store";
+import { IApplicationState } from "../../../../models/applicationState";
+import initialState from "../../../../redux/store/initialState";
 import ProjectSettingsPage, { IProjectSettingsPageProps } from "./projectSettingsPage";
 
 jest.mock("../../../../services/projectService");
 import ProjectService from "../../../../services/projectService";
+// jest.mock("../../../../redux/actions/projectActions");
```

Comment:
Lots of commented out test/mock stuff here - Just checking if this was intentional.